docs/server/spring-overview.md

```diff
@@ -7,6 +7,8 @@ title: Spring Server Overview
 is a Spring Boot auto-configuration library that automatically configures beans required to start up a reactive GraphQL
 web server.
 
+> NOTE: The server configuration is built on a [Spring WebFlux (reactive)](https://docs.spring.io/spring/docs/current/spring-framework-reference/web-reactive.html) stack. If you include Spring WebMVC or Spring Servlet, you may encouter issues.
```

Comment:
sentence above specifies that this "_is aSpring Boot auto-configuration library that automatically configures beans required to start up a **reactive GraphQL web server**_"

Reply:
Yea, I know. See this linked issue. Is there some way we can make that a little more clear?

Reply:
lets add a section on top with setup information, i.e. expand it with the info from https://medium.com/expedia-group-tech/creating-a-reactive-graphql-server-with-spring-boot-and-kotlin-54aca7316470